### DISCUSSION TITLE
Where does codeql extract the zip?

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#8455] [General]
When I open a zip database codeql prompts me it's unzipped:

![111](https://user-images.githubusercontent.com/1357701/158525629-ca683eb4-9a54-46f2-bf2c-d559b48eab0e.jpg)

I'm wondering where is it stored? And is the temporary path configurable?

**Comment by hvitved** [comment]
You can right-click on the database and click "Show Database Directory". I don't think it is possible to configure the temporary path, but if you instead unzip the DB manually in a desired location, then you can import the DB from there using "Choose Database from Folder".
